apps/dashboard/src/app/(dashboard)/dashboard/connect/layout.tsx

```diff
@@ -44,7 +44,7 @@ export default function DashboardConnectLayout(props: {
   children: React.ReactNode;
 }) {
   return (
-    <div className="flex flex-col md:flex-row md:h-full container px-0">
+    <div className="container flex flex-col px-0 md:flex-row md:h-full">
```

Comment:
@gregfromstl do u have some kind of linter for the class names, this looks like a linter did it. If u have one I want it what is it?